### DISCUSSION TITLE
Can't Add Team Mates

### DISCUSSION THREAD

**Discussion by GoodnatureBeacon** [AntlerVC/firetable#1031] [Support / Q&A] [answered]
Hey there, I just upgraded from Rowy V2, created a project and it all seems to have imported correctly. I then went to add some team mates to the project.

I got to workspace settings -> members -> invite user and it comes up with a pop-up that says "This workspace is on the free version of Rowy." And I can't add any user.

When I read the description of the free plan it says that it is suitable for individuals or small teams. Is this is a restriction of the plan? Or is something else going wrong here

**Comment by bofeiw** [comment, marked as answer]
Hi @GoodnatureBeacon, team collaboration is available on [Rowy Pro](http://rowy.io/pricing). You have 30 days free trial and can cancel anytime.
